### ISSUE TITLE
Missing stdlib docstrings in virtualenvs

### ISSUE THREAD

**Issue by iamkroot** [microsoft/pyright#1835]
**Describe the bug**
I'm seeing differing behaviour when running pyright inside a virtualenv, versus when using the default python interpreter. Specifically, the docstrings for objects from the standard library are not shown on hover when running inside a venv. 

Running 'Go to definition" on a builtin while using the default interpreter brings up the actual python file, which includes the docstrings. Consequently, running "hover" on it shows the docs _and_ the signature correctly annotated with types from typeshed (I assume, since the types aren't present in the source py file).
However, when inside a virtualenv, "Go to definition" directly jumps to the typeshed stub, and "hover" only shows the annotated signature without the docs. 

**To Reproduce**
First, disable any other LSP extensions (like the default Python package in VSCode, because it uses jedi to fill in the docs inside venvs).
Do the following twice, once using default python interpreter, and once with pyright configured to use a venv:
1. Import any standard library module written in python, like `os` or `threading`
2. Type out any of the user-facing functions, eg: `os.walk` or `threading.active_count`
3. Call "Hover"/"Show documentation" capability of pyright

**Expected behavior**
The docstrings for builtins should be available in both the modes - with and without virtualenvs.

**Screenshots or Code**
```python
import threading
threading.active_count
```
Default Python Interpreter:
![Default Python Interpreter](https://user-images.githubusercontent.com/29517049/117403523-f096d680-af25-11eb-823a-41a5d4d76ffc.png)

Virtualenv enabled:
![Virtualenv enabled](https://user-images.githubusercontent.com/29517049/117403528-f2f93080-af25-11eb-8a3d-1d9deac6ea25.png)

**VS Code extension or command-line**
Using the VSCode extension, v1.1.138

**Comment by erictraut** [comment]
I'm going to transfer this to the pylance-release issue list because this issue isn't related to the core type checking functionality in pyright.
